src/jalv.c

```diff
@@ -742,6 +749,7 @@ jalv_main(Jalv* jalv, int argc, char** argv)
 #ifdef HAVE_SUIL
 	suil_init(&argc, &argv, SUIL_ARG_NONE);
 #endif
+
 	if (jalv_init(&argc, &argv, &jalv->opts)) {
 		return -1;
```

Comment:
Call `jalv_close()` here to free backend.

Reply:
That seems incredibly likely to crash, since `jalv_close()` destroys all sorts of things that aren't even created at this point.

Reply:
Hmmm okay. May be we have to check in `jalv_close()` if the objects are not `NULL` and only destroy/free in this cases.
Or do you have another idea?
(I think always thinking about which objects has to be freed at a special error case could be a pain. Therefore I would prefer to nearly call `jalv_close()` on all errors)

Reply:
If you are fine with my idea I could spend some time to adapt `jalv_close()`